index.html

```diff
@@ -449,7 +449,7 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
         <p data-lang="zh-hant">減少字距，使得文字外框一部分重疊，稱作紧排。這種處理方式，主要應用於：</p>
         <ol>
           <li>
-            <p data-lang="en">For characters in headings in magazines or advertisements, reduced inter-character spacing can be used to keep the characters on one line, or it can also be used to achieve a special effect for presentation.</p>
+            <p data-lang="en">For characters in headings of magazines or advertisements, reduced inter-character spacing can be used to keep the characters on one line, or it can also be used to achieve a special effect for presentation.</p>
```

Comment:
Small nit: i think 'in' is correct if we are referring to headings of articles. 'Of' in 'in headings of magazines' sounds like the translation should have been 'of magazine titles', ie. the name of the magazine. Just a suggestion. Ignore if you wish.